update_screen.sh

```diff
@@ -56,17 +58,17 @@ while getopts ":a:d:e:" opt; do
     case $opt in
         a)
             scroll_up
-            add_target ${OPTARG}
+            add_target ${OPTARG} 
             print_targets
             ;;
         d)
             scroll_up
-            remove_target ${OPTARG}
+            remove_target ${OPTARG} $3
```

Comment:
>$3 [](start = 36, length = 2)

I cannot understand the usage of this script. Could you add a file usage comment block at the beginning?

Reply:
Added description for this change.